packages/fiori-elements-writer/src/index.ts

```diff
@@ -4,7 +4,7 @@ import { render } from 'ejs';
 import type { App, Package } from '@sap-ux/ui5-application-writer';
 import { generate as generateUi5Project } from '@sap-ux/ui5-application-writer';
 import { generate as addOdataService, OdataVersion, ServiceType } from '@sap-ux/odata-service-writer';
-import { generateOPAFiles } from '@sap-ux/ui5-test-writer';
+import { generateFioriElementsOPAFiles } from '@sap-ux/ui5-test-writer';
```

Comment:
Can we also keep `generateOPAFiles` for backward compatibility?
Fine if both names are exported `generateOPAFiles` && `generateFioriElementsOPAFiles` for the same fuction.